chainer/optimizer.py

```diff
@@ -376,6 +376,14 @@ def setup(self, link):
         Args:
             link (~chainer.Link): Target link object.
 
+        Returns:
+            The optimizer instance.
+
+        .. note::
+           As of v4.0.0, this function returns the optimizer instance itself
+           so that you can instantiate and setup the optimizer in one line,
+           e.g., ``optimizer = SomeOptimizer().setup(model)``.
```

Comment:
`.setup(link)` or `.setup(link=model)` would be more informative to me.

Reply:
Thanks, fixed as suggested.